includes/component/plugin-tracking/class-tyche-plugin-tracking.php

```diff
@@ -243,6 +243,11 @@ public function display_tracker_html_template() {
 				return;
 			}
 
+			$is_specific_page = apply_filters( $this->plugin_short_name . '_ts_tracker_display_notice', true );
```

Comment:
@PratapDungrani Please add this condition into the code block [here](https://github.com/TycheSoftwares/product-input-fields-for-woocommerce/blob/2d5b3e798cfa0af64b685381f3393ac569fb3d78/includes/component/plugin-tracking/class-tyche-plugin-tracking.php#L242). This way, we can have one code block that terminates the block execution.

Reply:
@PratapDungrani You also need to update this code change on the Plugin Tracking repo.